### DISCUSSION TITLE
How to disable some shortcuts

### DISCUSSION THREAD

**Discussion by erickRochaIP** [CodinGame/monaco-vscode-api#486] [Q&A] [answered]
Is there a way to disable shortcuts so that the user cannot access specific shortcuts? For example, I don't want the user to be able to use (Ctrl+,) and edit the User Settings.

**Comment by CGNonofr** [comment, marked as answer]
The settings edition and `Ctrl+,` keybinding comes with the preferences service override, you can remove it.

But it may not be enough to prevent the user from opening the settings file in a regular editor, you should also probably make that file readonly
